app/models/program.rb

```diff
@@ -5,4 +5,20 @@ class Program < ActiveRecord::Base
   validates :title, uniqueness: true
   validates :url, presence: true
   validates :url, uniqueness: true
+
+  def average
+    associated_reviews = reviews
+    submissions = associated_reviews.length
+    total_points = 0.0
+
+    associated_reviews.each do |review|
+      total_points += review.rating
+    end
+
+    if submissions == 0
+      0
+    else
+      (total_points / submissions).round(1)
```

Comment:
When we changed our code it was giving us NaN instead of 0 when there was no submissions.